src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -72,6 +72,15 @@
             <Setter Property="TextAlignment" Value="Center" />
             <Setter Property="VerticalAlignment" Value="Center"/>
         </Style>
+        <Style x:Key="customButtonFocusVisualStyle">
+            <Setter Property="Control.Template">
+                <Setter.Value>
+                    <ControlTemplate>
+                        <Rectangle Margin="1" Stroke="White" StrokeDashArray="1 2" StrokeThickness="1" />

```

Comment:
nit: Just of curiosity, we could check strokeThickness=2, for even better visibility..

Reply:
I’ve put thickness to 1 and dash case to mimic appearance of original (default) focus rectangle. However we can put whatever we want. I played with it a bit, and personally I like full line (without dashes) eve more. It’s really visible and it looks better. We can go with any style we like as long as color contrast is fulfilled.

Reply:
Can you post a screenshot with the full line?

Reply:
On second thoughts, we should not make the change in this PR since it will also require to change it for several other controls. Please open an issue to track it. Thanks.

Reply:
Sure, I’ll merge this and open follow up ticket.